lib/ts/recipe/session/index.ts

```diff
@@ -325,44 +334,62 @@ export default class SessionWrapper {
             userContext,
         });
     }
-    static revokeAllSessionsForUser(userId: string, userContext: any = {}) {
-        return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.revokeAllSessionsForUser({ userId, userContext });
+    static revokeAllSessionsForUser(userId: string, tenantId?: string, userContext: any = {}) {
+        return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.revokeAllSessionsForUser({
+            userId,
+            tenantId: tenantId === undefined ? DEFAULT_TENANT_ID : tenantId,
+            userContext,
+        });
     }
 
-    static getAllSessionHandlesForUser(userId: string, userContext: any = {}) {
+    static getAllSessionHandlesForUser(userId: string, tenantId?: string, userContext: any = {}) {
         return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.getAllSessionHandlesForUser({
             userId,
+            tenantId: tenantId === undefined ? DEFAULT_TENANT_ID : tenantId,
             userContext,
         });
     }
 
-    static revokeSession(sessionHandle: string, userContext: any = {}) {
-        return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.revokeSession({ sessionHandle, userContext });
+    static revokeSession(sessionHandle: string, tenantId?: string, userContext: any = {}) {
+        return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.revokeSession({
+            sessionHandle,
+            tenantId: tenantId === undefined ? DEFAULT_TENANT_ID : tenantId,
+            userContext,
+        });
     }
 
-    static revokeMultipleSessions(sessionHandles: string[], userContext: any = {}) {
+    static revokeMultipleSessions(sessionHandles: string[], tenantId?: string, userContext: any = {}) {
```

Comment:
This also means, that in the core:
- We change the plugin interface functions to accept an AppIdeitifier instead of a TenantIdetifier for these functions
- The db queries will change
- We change the db schema for session_info table to make the primary key `(app_id, session_handle)`.
- We also need to update the migration script.
- We also need to update the CDI (patch version).
- We also need to update the core version to 6.0.1.
- So in the core changes, we need to make the primary change automatically if not done already (cause we are releasing as a patch version change).

Reply:
Done